src/ff/prime_field.rs

```diff
@@ -25,41 +28,40 @@ impl<F: PrimeField> Serializable for F {
 }
 
 macro_rules! field_impl {
-    ( $field:ident, $int:ty, $prime:expr, $arraylen:ty ) => {
```

Comment:
Same as the `GaloisField` macro. `$arraylen` is now defined in `Block` so no need to pass it here.